src/gen.py

```diff
@@ -225,6 +225,11 @@ def parse_options(config: 'fbm.PackageGeneratorConfig'):
         help="Blender version for specific mod patches to be applied "
              "(ex. 2.79, 2.80)"
     )
+    parser.add_argument(
+        "-g", dest="generated_mods_dir", type=str,
+        help="generated mod patches directory"
+             "(ex. src/mods/generated_mods)"
+    )
```

Comment:
We don't want to add a redundant option.
Could you consider to remove this option?

Reply:
`src/mods/generated_mods` is a temporary directory at build time, so it seems inappropriate to put it in `src`.

I offer two solutions.
1. Manage temporary directory lifecycle (create & remove) inside gen.py and add a keep option for debug
2. Manage temporary directory lifecycle (create & remove) outside gen.py and pass it as an option (this PR)

What do you think?

Reply:
solution 3 is good for now.

But, if you want to pass an option, it is better to pass the whole `mods` directory instead of `generated_mods`.
In the future, I will not to distinguish between `generated_mods` and the others.

Reply:
Thanks for your consideration.
I will edit the pull request for solution 3.

Reply:
Pull request editing is complete.
Please review again.